engine/src/main/java/org/hibernate/search/v6poc/backend/document/spi/UserIndexFieldConverter.java

```diff
@@ -22,26 +23,41 @@
 public final class UserIndexFieldConverter<F> {
 
 	private final ToIndexFieldValueConverter<?, ? extends F> dslToIndexConverter;
+	private final FromIndexFieldValueConverter<? super F, ?> projectionFromIndexConverter;
 
-	UserIndexFieldConverter(ToIndexFieldValueConverter<?, ? extends F> dslToIndexConverter) {
+	UserIndexFieldConverter(ToIndexFieldValueConverter<?,? extends F> dslToIndexConverter,
+			FromIndexFieldValueConverter<? super F,?> projectionFromIndexConverter) {
 		this.dslToIndexConverter = dslToIndexConverter;
+		this.projectionFromIndexConverter = projectionFromIndexConverter;
 	}
 
 	@Override
 	public String toString() {
 		return getClass().getName() + "["
 				+ "dslToIndexConverter=" + dslToIndexConverter
+				+ ",projectionFromIndexConverter=" + projectionFromIndexConverter
```

Comment:
Minor but don't we have a space after the comma elsewhere?

Reply:
Seems we do, most of the time. Fixed.